spinnaker/resource_application.go

```diff
@@ -88,32 +91,34 @@ type Permissions struct {
 	Write   []string `json:"WRITE"`
 }
 
-func resourceSpinnakerApplicationCreate(d *schema.ResourceData, meta interface{}) error {
+func resourceSpinnakerApplicationCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
```

Comment:
[note]
[Support for Diagnostics](https://www.terraform.io/docs/extend/guides/v2-upgrade-guide.html#support-for-diagnostics)

> A number of new functions have been added that return a diag.Diagnostics type. This type can be used to return multiple errors and warnings to Terraform, and to associate those errors or warnings with specific fields.